doc/mgr/administrator.rst

```diff
@@ -31,6 +31,52 @@ which should now include a mgr status line::
 
     mgr active: $name
 
+Interpreting Manager Daemon Status
+==================================
+
+A cluster's health status will show each ``ceph-mgr`` daemon in one of three states:
+
+1. **active**
+
+   This manager daemon has been fully initialized, which means it is ready to receive
+   and execute commands. Only one manager will be in this state at a time.
+
+2. **active (starting)**
+
+   This manager daemon has been chosen to be ``active``, but it is not done initializing.
```

Comment:
Nit: There’s a bit of a trend to capitalize Manager and Monitor. Here and below.